sunpy/map/sources/hinode.py

```diff
@@ -98,6 +98,15 @@ def processing_level(self):
             return
         return int(lvl)
 
+    @property
+    def unit(self):
+        unit = "DN"
+        response = self.meta.get('HISTORY')
+        for i in response:
+            if i[:15] == "XRT_RENORMALIZE":
+                unit = "DN/sec"
```

Comment:
Does this not have to be an astropy unit or does a string work?

Reply:
yes totally, thanks for pointing out